Frends.Radon/RadonProgram.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Linq;
+
+namespace Frends.Radon
+{
+    public class RadonProgram
+    {
+        private readonly IRadonExecutor _executor;
+
+        public RadonProgram(IRadonExecutor executor)
+        {
+            _executor = executor;
+        }
+
+        public RadonResult ExecuteRadon()
```

Comment:
This is probably redundant now? The `Reporter` task could just call RadonExecutor directly...